prompts.py

```diff
@@ -100,5 +95,22 @@
 """, 1000, None, 5, check_proof, ['Dafny'])
 
 # Set the right-hand side to the selected problem.
-(prompt, max_new_tokens, expansion_count, min_lines, check_fun, supported_langs) = problem_intersperse_dafny
+(prompt, max_new_tokens, expansion_count, min_lines, check_fun, supported_langs) = problem_fact
 assert lang in supported_langs
+if lang == 'Lean4':
+    prompt += '''
```

Comment:
This is adding the string to the end of the prompt: we can do `prompt = prompt.replace(f"### {lang}:", custom + f"\n### {lang}:")`? Or keeping it where it was.

Reply:
Or just `prompt = custom + "\n" + prompt?`

Reply:
Ok!

Thank you!